jwt-authorizer/tests/integration_tests.rs

```diff
@@ -145,7 +143,8 @@ async fn make_proteced_request(app: &mut Router, bearer: &str) -> Response {
 }
 
 async fn make_public_request(app: &mut Router) -> Response {
-    app.ready()
+    app.as_service()
+        .ready()
         .await
         .unwrap()
         .call(Request::builder().uri("/public").body(Body::empty()).unwrap())
```

Comment:
Here I think you could call `call` directly on as_service() (type `RouterAsService`).